docs/source/photonics/examples/waveguide_modes.py

```diff
@@ -25,8 +25,14 @@
 )
 
 with tempfile.TemporaryDirectory() as tmpdirname:
-    mesh_from_OrderedDict(polygons, resolutions, filename=tmpdirname + '/mesh.msh', default_resolution_max=10)
-    mesh = Mesh.load(tmpdirname + '/mesh.msh')
+    mesh_from_OrderedDict(
+        polygons,
+        resolutions,
+        filename=f'{tmpdirname}/mesh.msh',
+        default_resolution_max=10,
+    )
+
```

Comment:
```suggestion
    mesh_from_OrderedDict(polygons, resolutions, filename=f'{tmpdirname}/mesh.msh', default_resolution_max=10)

```
The usage of f-strings is nice! I don't see the point in adding the line breaks here.